db_stress_tool/no_batched_ops_stress.cc

```diff
@@ -243,20 +292,43 @@ class NonBatchedOpsStressTest : public StressTest {
 
     for (const auto& s : statuses) {
       if (s.ok()) {
-        // found case
-        thread->stats.AddGets(1, 1);
+#ifndef NDEBUG
+        if (fault_fs_guard && error_count && !filter_read_error) {
+          // Grab mutex so multiple thread don't try to print the
+          // stack trace at the same time
+          MutexLock l(thread->shared->GetMutex());
+          fprintf(stderr, "Didn't get expected error from MultiGet\n");
+          fprintf(stderr, "Callstack that injected the error\n");
+          fault_fs_guard->PrintFaultBacktrace();
```

Comment:
Now I get that, different `status` can have different stack trace when errors occur.